src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -1297,16 +1297,9 @@ public JSONWebResponse postJSON(@NonNull String path, @NonNull Object json) thro
             }
 
             WebResponseData webResponseData;
-            InputStream responseStream = conn.getInputStream();
```

Comment:
[`conn.getInputStream()`](https://docs.oracle.com/javase/8/docs/api/java/net/URLConnection.html#getInputStream--) will never return `null` - unless some code is violating the API contract.
Thus leading to spotbugs complaining about `RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE` 

this rest of the diff moves to `try-with-resources` to cleanup the code knowing that the null check is not needed.